checkstyle/suppressions.xml

```diff
@@ -540,6 +540,8 @@
     <suppress checks="ClassFanOutComplexityCheck" files="com.hazelcast.map.impl.proxy.MapProxySupport"/>
     <suppress checks="ClassDataAbstractionCoupling" files="com.hazelcast.map.impl.proxy.MapProxySupport"/>
     <suppress checks="CyclomaticComplexity" files="com.hazelcast.map.impl.client.AbstractTxnMapRequest"/>
+    <suppress checks="MethodCount|ClassDataAbstractionCoupling" files="com.hazelcast.map.impl.MapServiceContextImpl"/>
+    <suppress checks="MethodCount" files="com.hazelcast.map.impl.MapServiceContext"/>
```

Comment:
Probably the MapServiceContext interface can be collapsed in the MapServiceContextImpl and all the getters can be removed and just allow for public field access. Most of the methods on the MapServiceContext are just dependency getters.